api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -178,6 +178,11 @@
         org.pmiops.workbench.model.CohortReview responseReview = TO_CLIENT_COHORTREVIEW.apply(cohortReview);
         responseReview.setParticipantCohortStatuses(paginatedPCS.stream().map(TO_CLIENT_PARTICIPANT).collect(Collectors.toList()));
 
+        responseReview.setPage(PAGE);
+        responseReview.setPageSize(LIMIT);
+        responseReview.setSortOrder("ASC");
```

Comment:
Can we create an enum here instead of using a String? Something like:
SortType:
    type: string
    description: possible sort order
    enum: [ASC, DESC]

then you can do something like this: 
responseReview.setSortOrder(SortType.ASC);

Reply:
As discussed offline, we're punting this for future consideration